platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegion.java

```diff
@@ -276,6 +295,11 @@ public void run() {
               if (observer != null) {
                 observer.onError(error);
               }
+              try {
+                telemetry.onOfflineDownloadEndFailure(definition, error);
```

Comment:
This error event is invoked for each individual download failure of resources in the offline region. It may be called multiple times. No single error event represents an end to the download or a failed download of the entire region.